ui/app/templates/components/auth-info.hbs

```diff
@@ -30,27 +31,25 @@
               </button>
             </li>
             <li class="action">
-              <ConfirmAction
-                @buttonClasses="button link is-destroy"
-                @confirmTitle={{concat "Revoke " (get auth 'authData.displayName') "?"}}
-                @confirmMessage={{concat "You will not be able to log in again with this token."}}
-                @confirmButtonText="Revoke"
-                @onConfirmAction={{action "revokeToken"}}
-              >
-                Revoke token
-              </ConfirmAction>
+              <c.Message
+                @id={{get auth 'authData.displayName'}}
+                @title={{concat "Revoke " (get auth 'authData.displayName') "?"}}
+                @onConfirm={{action "revokeToken"}}
+                @message="You will not be able to log in again with this token."
+                @triggerText={{concat "Revoke " (get auth 'authData.displayName')}}
```

Comment:
Isn't it the trigger that should say `Revoke token`? An the confirm button should say `Revoke`? I think showing the displayName here may be misleading because we're revoking the token, not acting on the entity.

Reply:
that is 1000% correct, thank you 🙏